src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,75 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+type Props = {
+  deleteCompleted: () => void,
+  activeTodos: Todo[],
+  completedTodos: Todo[],
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  deleteCompleted,
+  activeTodos,
+  completedTodos,
+}) => {
+  return (
+    <footer className="footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${activeTodos.length} items left`}
+      </span>
+
+      <ul className="filters">
+        <li>
+          <NavLink
+            to="/"
+            className={({ isActive }) => classNames(
+              {
+                selected: isActive,
+              },
+            )}
+          >
+            All
+          </NavLink>
+        </li>
+
+        <li>
+          <NavLink
+            to="/active"
+            className={({ isActive }) => classNames(
+              {
+                selected: isActive,
+              },
+            )}
+          >
+            Active
+          </NavLink>
+        </li>
+
+        <li>
+          <NavLink
+            to="/completed"
+            className={({ isActive }) => classNames(
+              {
+                selected: isActive,
+              },
+            )}
+          >
+            Completed
+          </NavLink>
+        </li>
+      </ul>
+
+      {(completedTodos.length)
+        ? (
+          <button
+            type="button"
+            className="clear-completed"
+            onClick={() => deleteCompleted()}
+          >
+            Clear completed
+          </button>
+        )
+        : null}
```

Comment:
use `&&` instead of ternary (as far as you return `null`)
for example
```
{completedTodos.length && {
  ...
}}
```
it works the same way but looks more consistent

Reply:
thanks, you are totally right.